app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -57,7 +57,7 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
         )}
         {experiment.targetingConfigSlug && (
           <tr>
-            <th>Custom audience</th>
+            <th>Advanced Targeting</th>
```

Comment:
Nit - gonna wanna update this describe block: https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.test.tsx#L90:L90